src/es/components/molecules/carouselTwo/CarouselTwo.js

```diff
@@ -665,17 +743,37 @@ export default class CarouselTwo extends Mutation() {
     })
   }
 
-  previous (focus) {
+  previous(focus) {
     if (this.getAttribute('namespace') === 'carousel-two-teaser-') return this.scrollIntoView((this.activeSlide && this.activeSlide.previousElementSibling.previousElementSibling) || Array.from(this.section.children)[this.section.children.length - 1], focus)
+    if (this.getAttribute('namespace') === 'carousel-two-3-column-')
+      return this.scrollIntoView(
+        (
+          this.activeSlide &&
+          this.activeSlide.previousElementSibling &&
+          this.activeSlide.previousElementSibling.previousElementSibling &&
```

Comment:
Das könnte man mit optional chaining vereinfachen oder?
Der folgende Codeblick ist auch generell etwas schwer nachzuvollziehen, vielleicht kann man da noch mit variablen was vereinfachen oder wenigstens einen Kommentar rein setzen, was hier passiert :)

Reply:
Done. Added some Comments and refactored logic to make it more clear what is going on